### PR TITLE
Move the files instead of copying when ingest SST files

### PR DESCRIPTION
**Pull request by darionyaphet**

### CODE REVIEW

File: src/kvstore/RocksEngine.cpp [hunk 1]
```diff
@@ -330,6 +330,7 @@ int32_t RocksEngine::totalPartsNum() {
 
 ResultCode RocksEngine::ingest(const std::vector<std::string>& files) {
     rocksdb::IngestExternalFileOptions options;
+    options.move_files = true;
```
Comment (liuyu85cn) [inline src/kvstore/RocksEngine.cpp:333]: how about use a gflag here?

Comment (monadbobo) [reply src/kvstore/RocksEngine.cpp:333]: > how about use a gflag here?

I agree with you, or add a parameter for ingest.

**Review by liuyu85cn — DISMISSED** [review]

File: src/kvstore/RocksEngine.cpp [hunk 2]
```diff
@@ -336,6 +339,9 @@ int32_t RocksEngine::totalPartsNum() {
 
 ResultCode RocksEngine::ingest(const std::vector<std::string>& files) {
     rocksdb::IngestExternalFileOptions options;
+    if (FLAGS_move_files) {
+        options.move_files = true;
+    }
```
Comment (bright-starry-sky) [inline src/kvstore/RocksEngine.cpp:344]: What's about `options.move_files = FLAGS_move_files` ?

Comment (darionyaphet) [reply src/kvstore/RocksEngine.cpp:344]: Good ~

**Review by bright-starry-sky — APPROVED** [review]
Good job. LGTM

**Review by liuyu85cn — APPROVED** [review]

**Pull request merged**